ember-cli-build.js

```diff
@@ -1,10 +1,10 @@
-/* eslint-env node */
 'use strict';
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function (defaults) {
-  const app = new EmberApp(defaults, {
+  let app = new EmberApp(defaults, {
```

Comment:
This was automatically changed by the blueprint updates.

I see there's not a general consensus in the Ember community of which one to use (some big addon maintainers apparently prefer "liberal let" instead of "constantly const"). 
The default ESLint configuration that comes with the Ember app blueprint does not include any rule that enforces one or the other.

What should we do here?

* We could allow both (Ember default app blueprint does that).
* We could configure ESLint's `prefer-const` and keep reverting these automatic conversions by `ember-cli-update`.
* We could install [some ESLint plugin](https://www.npmjs.com/package/eslint-plugin-prefer-let) to enforce let.

Reply:
eh, I think option 1 is best then, it doesn't really matter that much if it's this much of a discussion in the Ember community. I am just surprised by it, because I am used to getting bashed for not using cost instead of let whenever possible

Reply:
Ah yes I was one of those people 😂 apologies